{"uuid": "f8c5e9f0-47c5-489d-9a6e-2ae15e972715", "children": ["22087b99-71f9-4a8c-aef9-e553daa322a3", "3dbf6d3c-2f7b-4db0-9441-62e2e91c472d", "a433f459-4468-414c-9c1a-d4921c095768", "f90107b4-4f4f-475a-b0d7-5a1fb0f0d813", "4a553aa7-8667-43b3-910f-bade4d350e29", "fd4b9a3e-aaa0-4ce6-b8f1-9d82d68b20e3", "34145695-7373-493a-aba3-f0bf07dd9d78", "b6ff7357-799e-4a6e-8b0d-b324205bc477", "9948670e-0524-481f-b256-6ef2e3f5a9dd"], "befores": [{"name": "get_job_details", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy-1.5.0-py3.10.egg/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_API_Suite/test_monefy.py\", line 16, in get_job_details\n    peviitor_jobs_data = testutils.scrape_peviitor(company_name, 'Rom√¢nia')\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 70, in scrape_peviitor\n    response_data = TestUtils._get_request(params)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 48, in _get_request\n    updateapi.get_token()\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/__utils/peviitor_update.py\", line 42, in get_token\n    self.access_token = requests.request(\"POST\", \"https://api.peviitor.ro/v5/get_token/\", headers=post_header, data=payload).json()['access']\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/models.py\", line 975, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1714147090177, "stop": 1714147094752}], "start": 1714147090177, "stop": 1714147095196}